src/hardware/input/mouseif_dos_driver.cpp

```diff
@@ -60,7 +60,7 @@ static constexpr uint8_t  cursor_size_x  = 16;
 static constexpr uint8_t  cursor_size_y  = 16;
 static constexpr uint16_t cursor_size_xy = cursor_size_x * cursor_size_y;
 
-static constexpr uint8_t num_buttons = 3;
+static constexpr uint8_t MaxButtons = 3;
 
 enum class MouseCursor : uint8_t { Software = 0, Hardware = 1, Text = 2 };
```

Comment:
If you need it to be `uint8_t` for binary "enum compatibility" with DOS drivers / interuppts, it's fine, otherwise just remove the width specifier if it's 100% native code.

Reply:
Changed to `auto` - this particular value is not returned to the guest side.